lib/src/mock-server/server.ts

```diff
@@ -33,6 +33,7 @@ export function runMockServer(
   }
 ) {
   const app = express();
+  app.use(express.raw({ type: () => true }));
```

Comment:
This is to force `express` to always populate the `request.body` attribute, event on `GET` requests. Before this change, received `GET` requests with bodies did had a `undefined` `request.body`. This meant that the request body wasn't being proxied through, causing the upstream to reject the request or hang.